src/unstract/adapters/embedding/hugging_face/src/hugging_face.py

```diff
@@ -56,25 +54,15 @@ def get_embedding_instance(self) -> Optional[BaseEmbedding]:
             embedding_batch_size = EmbeddingHelper.get_embedding_batch_size(
                 config=self.config
             )
-            if (self.config.get(Constants.MAX_LENGTH)) is not None:
-                embedding = HuggingFaceEmbedding(
-                    model_name=str(self.config.get(Constants.MODEL)),
-                    tokenizer_name=str(
-                        self.config.get(Constants.TOKENIZER_NAME)
-                    ),
-                    normalize=bool(self.config.get(Constants.NORMALIZE)),
-                    embed_batch_size=embedding_batch_size,
-                    max_length=int(self.config.get(Constants.MAX_LENGTH, 10)),
-                )
-            else:
-                embedding = HuggingFaceEmbedding(
-                    model_name=str(self.config.get(Constants.MODEL)),
-                    tokenizer_name=str(
-                        self.config.get(Constants.TOKENIZER_NAME)
-                    ),
-                    normalize=bool(self.config.get(Constants.NORMALIZE)),
-                    embed_batch_size=embedding_batch_size,
-                )
+            max_length: Optional[int] = self.config.get(Constants.MAX_LENGTH)
+            embedding: Optional[BaseEmbedding] = HuggingFaceEmbedding(
+                model_name=str(self.config.get(Constants.MODEL)),
+                tokenizer_name=str(self.config.get(Constants.TOKENIZER_NAME)),
+                normalize=bool(self.config.get(Constants.NORMALIZE)),
+                embed_batch_size=embedding_batch_size,
+                max_length=max_length,
+            )
```

Comment:
NIT: Similar to how you initialize the other values, you can directly pass max_length
```suggestion
            embedding: Optional[BaseEmbedding] = HuggingFaceEmbedding(
                model_name=str(self.config.get(Constants.MODEL)),
                tokenizer_name=str(self.config.get(Constants.TOKENIZER_NAME)),
                normalize=bool(self.config.get(Constants.NORMALIZE)),
                embed_batch_size=embedding_batch_size,
                max_length=int(self.config.get(Constants.MAX_LENGTH)),
            )
```

Reply:
@chandrasekharan-zipstack  max_length is defaulted to None if not passed inside the HuggingFace constructor. It is also an optional param. If user is not feeding it in, better to set it as None as defaulted to in Hugging Face. Retaining the variable max_length for readability and less clutter.